app.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask, jsonify
+import json
+from all_recipe_retriever import retrieve
+from my_encoder import MyEncoder
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello():
+    return 'Hello World!'
+
+
+@app.route('/recipes/')
+def get():
+    """
+        returns a list of recipes
+        """
+    recipes = retrieve()
+    print("recipes retrieved")
+    print(recipes)
+    data = []
+    for recipe in recipes:
+        data.append(MyEncoder().encode(recipe))
```

Comment:
In this case we prefer the list comprehension : [](https://python.doctor/page-comprehension-list-listes-python-cours-debutants)
```suggestion
        data = [
           MyEncoder().encode(recipe)
           for recipe in recipes
        ]
```

Reply:
done